### PR TITLE
Project #6 Newsweek clone on the newsweek branch

### PR DESCRIPTION
**Pull request by mke2111**
Newsweek clone page designed with HTML, CSS, and Bootstrap to replicate the original page for most if not all of the details.

We used semantic HTML and most of the CSS best practices as possible.

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,565 @@
...
+  padding: 0 0;
+}
+
+body {
+  background-color: #f4f5f6;
+}
+
+ul {
+  list-style: none;
+}
```
Comment (ghost) [inline styles/styles.css:14]: This can be implemented using bootstrap `<ul class="list-unstyled">`.

File: styles/styles.css [hunk 2]
```diff
@@ -0,0 +1,565 @@
...
+h4,
+h5,
+h6 {
+  line-height: 1.1;
+  font-family: 'Roboto Condensed', sans-serif;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
```
Comment (ghost) [inline styles/styles.css:29]: `<a href="#" class="text-decoration-none">`.

File: styles/styles.css [hunk 3]
```diff
@@ -0,0 +1,565 @@
...
+  border-bottom: 1px solid #f72210;
+}
+
+ul li {
+  display: inline-block;
+  margin: 0;
+  padding: 0;
+}
```
Comment (ghost) [inline styles/styles.css:42]: You can use this bootstrap styling `<li class="d-none m-0 p-0">` for line 36-40.

Comment (ghost) [reply styles/styles.css:42]: The above are a few examples. Most of your CSS styling could easily be done by bootstrap. Please review it.

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required

## Hello team,

**Great job on your project.**

**Please make the following changes.**

1). **There is some content that is missing above the date on the navbar.**

original | clone
---------| -------
date_original | date_clone

2). **The login text on the navbar should be a link.**

3). **The navbar should have a sticky position on all screen sizes.**

4). **There is a lot of empty space on the left and right sides of your body when compared to the original page. Hint: The container class has a big margin-left and margin-right. You can try to reduce the margin values.**

original | clone
-------- | -------
container_original | container_clone

5). **Please insert the image tags on the left bottom part of the image as shown in the original.**

original | clone
---------| -------
cultural_original | cultural_clone

6). **Ensure all the contents in the mid-nav display on one line at 1024px width.** 

original | clone
---------| -------
midnav_original | midnav_clone

7). **Add space between the main section and the header section, at 768px width. The featured stories and top story titles are hidden by the navbar.**

8).  **Add space between the main section and the header section, at 767px width. The top story title is hidden by the navbar.**

9). **The Newsweek logo in the footer should be a link.**

### Since this is a bootstrap project, you are required to use bootstrap styling instead of custom styling. Only apply custom styling if bootstrap styling is not applicable ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/using_bootstrap/bootstrap_classes_requirement.png) from pathwright). Please check the comments below for more on this.

### Happy coding 😃.

**PS: You can reach me on slack(Rose Wanjohi).**

File: README.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+
+![screenshot](./newsweek-screenshot.png)
+
+
+## Built With
+
+- HTML
+- CSS
+
```
Comment (javitocor) [inline README.md]: Add bootstrap in your readme, you will want the recruiters to know that you can use this tech

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Hello team!
Good job implementing the changes! 👍 
You are so close, just some more small changes and ready to go! 💪 

Please, check my comments below 👇  apply the changes and resubmit your project tfor a new code review. ⏩ 

Good luck with it! 😃 

## Changes

- Navbar should be sticky or fixed for all screen sizes.
- The navbar should take the whole width of the screen
- In the magazine section the first image has a different size than the others, kindly modify it
Capture1
Capture
- The subscribe section must have 2 images, please add one.
Capture2

Capture3

- For screens under 768px, the order of the sections must be: top story, featured stories, the debate, more stories, my turn, culture&travel. Kindly, make your project like the original.

[OPTIONAL]
Since this is a project to use and learn to use bootstrap, maybe you can try to reduce your CSS, now it is more than 500 lines, try to make it around 400.

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

#### Hello @Janluc  and @mke2111, Great job! 👏  

You've done an awesome job so far 👌. But there are still some changes that are necessary for this project to be approved.

In this project, we have to make sure nothing is messed up, please kind fixed the following and you should be good to go.

Please refer to the screenshot below:

- [ ] Let your navbar have a fullwidth on the mobile view

- [ ] Fix the overlapping between the 'subscription offers' and the hamburger menu

- [ ] The width of your page should scale to 100%

- [ ] The Newsweek logo is almost gone, kindly fix it to a proper size

- [ ] The font-color of the footer menu is too faint, kindly fix this too

mobile

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Good effort with the requested changes, team! 👍 
Your project looks amazing! ❤️ 
You are so close!
A few more changes and ready to go! ⏩ 
Kindly, check my comments below 👇 

## Changes

- Navbar should be sticky also in big screens, please make it happen
- We are having horizontal scrollbar when the screen is under 1050px, please remove it
- Mobile screens under 768px:
   - The navbar still have some gap on the right side, kindly make it 100% width.
   - The order of the sections must be: top story, featured stories, the debate, opinion, more stories, my turn, culture&travel. Please, make your project to have the same order
- Please fix the signup link
Capture1
Capture
- Kindly reduce your lines of CSS, this is a project to use bootstrap framework, so you have to learn to use every class, so more framework and less CSS.

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Team, good job with the changes made so far! You are so close! 👍 
Two more things and ready to go! 💪 

## Changes

- The navbar should be fixed and 100% width for all screens, please make it happen, on large screens still is not 100% width and not yet fixed.
- For screens below 768px, the order of sections still does not correspond with the original, after 'opinion' comes 'more stories', 'my turn' and 'culture&travel', in that order. Kindly modify it to make it equal to the original

**Review by jubaan — APPROVED** [review]
# Status: Approved :tada: :tada: :tada:

You showed effort and have done great with the project!

## Congrats team you may now merge to master!

## See you at the top :rocket: !

**Pull request merged**